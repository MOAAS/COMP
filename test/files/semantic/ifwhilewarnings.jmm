class Simple {
    public int fun(int a, int b) {
        while (true) {

        }

        while (false) {

        }

        if (true) {

        }
        else if (false) {

        }
        else {

        }

        a = a / 0;

        return 0;


    }
 }