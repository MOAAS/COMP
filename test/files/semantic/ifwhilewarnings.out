Warning: Division by zero at line 21, column 17.
Warning: While expression will always be true at line 3, column 15.
Warning: While expression will always be false at line 7, column 15.
